ui/stashhelp.go

```diff
@@ -100,7 +100,6 @@ func (m stashModel) helpView() (string, int) {
 	}
 
 	var (
-		isEditable    bool
```

Comment:
In the code as it is in `master`, this variable was simply never set. It is a left-over from previous changes.

In the past, it was set [like this](https://github.com/charmbracelet/glow/pull/442/files#diff-8d8107f924235e9ef0d2002f9b3088317067bccace395463b8bb62c3dcf8cf2cR126);

```go
isEditable = md != nil && md.docType == LocalDoc && md.localPath != ""
```

That's not easily addable to our current code because it was refactored since.

Question: In which situation a file would not be editable? Is it fine to always show the hint?

Reply:
cc @caarlos0 @aymanbagabas